Python/Product/BuildTasks/Microsoft.PythonTools.targets

```diff
@@ -200,36 +200,20 @@ permissions and limitations under the License.
   <Target Name="CreateManifestResourceNames" />
   <Target Name="_CopyAppConfigFile" />
 
-  <PropertyGroup>
-    <BuiltProjectOutputGroupDependsOn>
-      BuiltProjectOutputGroupFast;
-      $(BuiltProjectOutputGroupDependsOn)
-    </BuiltProjectOutputGroupDependsOn>
-  </PropertyGroup>
-  
-  <Target Name="BuiltProjectOutputGroupFast"
-          Returns="@(BuiltProjectOutputGroupFastOutput)"
-          DependsOnTargets="$(BuiltProjectOutputGroupFastDependsOn)">
+  <Target Name="BuiltProjectOutputGroup"
+          Returns="@(BuiltProjectOutputGroupOutput)"
+          DependsOnTargets="$(BuiltProjectOutputGroupDependsOn)">
     <ItemGroup>
-      <BuiltProjectOutputGroupFastOutput Remove="@(BuiltProjectOutputGroupFastOutput)" />
-      <_BuiltProjectOutputGroupFastOutput Remove="@(_BuiltProjectOutputGroupFastOutput)" />
+      <BuiltProjectOutputGroupFastOutput Remove="@(BuiltProjectOutputGroupOutput)" />
+      <_BuiltProjectOutputGroupFastOutput Remove="@(_BuiltProjectOutputGroupOutput)" />
```

Comment:
These 2 'FastOutput' are the only references left to Fast in the file. Is that correct?